frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -155,6 +159,17 @@ local footerTextGeneratorMap = {
             return BD.wrap(prefix) .. " " .. (powerd:isCharging() and "+" or "") .. batt_lvl .. "%"
         end
     end,
+    bookmark_count = function(footer)
+        local symbol_type = footer.settings.item_prefix or "icons"
+        local prefix = symbol_prefix[symbol_type].bookmark_count
+        --retrieve bookmark count:
+        local bookmark_count = footer.ui.bookmark:getNumberOfBookmarks()
+        -- if no bookmarks defined, don't show icon:
+        if bookmark_count == 0 then
+            return ""
+        end
+        return prefix .. tostring(bookmark_count)
```

Comment:
You forgot a space between the prefix and number
```suggestion
        return prefix .. " " .. tostring(bookmark_count)
```

Reply:
No, I hadn't forgotten that, it was a choice of mine to make this item as compact as possible (my footer has many items, so every space extra comes at a cost). I actually like it better without the extra space.

If more people support this extra space, I am allright with adding it.

Reply:
Of course I don't think this one specific item should randomly be different.

I will point out that there exist such things as thin spaces. It's not a matter of all or nothing. (In the space sense. It's all or nothing in the consistency sense.)

https://commons.wikimedia.org/wiki/File:Punctuation-Spaces.svg

Reply:
Thin spaces? How to accomplish in KOReader? Also with a svg? Seems unlikely to me.

I understand the argument for consistency, but still think the item looks much nicer without the space.

But as I said, then I'll insert it (and remove it in my personal version 😄 ).

Reply:
lol no, you'd use it in the string: ` `

But not in this PR please. ;-)

Reply:
I don't understand: it _was_ the intention that I'd add the space in this PR? Then I've misunderstood. Where do you want me to add the extra space?

Reply:
The space is good now, all's fine in this PR as far as I'm concerned.

But changing the space to a thin space or a hair space or something — that should be a different PR.

Reply:
Yes, I think it is best to leave things as they are now.